#! /bin/sh /usr/share/dpatch/dpatch-run
## no_developer_docs.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not build developer's documentation.

@DPATCH@
diff -urNad linphone-1.3.5~/Makefile.am linphone-1.3.5/Makefile.am
--- linphone-1.3.5~/Makefile.am	2006-04-04 08:18:47.000000000 +0000
+++ linphone-1.3.5/Makefile.am	2006-05-15 09:34:52.000000000 +0000
@@ -7,7 +7,7 @@
 
 SUBDIRS = m4 support pixmaps po ipkg oRTP gsmlib lpc10-1.5 \
                 $(WIN32ACM_DIR) mediastreamer mediastreamer2\
-		media_api exosip coreapi console gnome share developer-docs
+		media_api exosip coreapi console gnome share
 
 
 EXTRA_DIST =  linphone2.glade BUGS linphone.kdevprj \
